iml-agent/src/action_plugins/stratagem/action_filesync.rs

```diff
@@ -40,8 +40,9 @@ fn do_rsync<'a>(
         .chunks(10)
         .map(Ok)
         .try_fold(vec![], |mut acc, xs| async {
-            let xs = try_join_all(xs).await?;
+            let mut xs = try_join_all(xs).await?;
 
+            xs.retain(|x| x.errno != 0);
             acc.extend(xs);
```

Comment:
@jgrund 
Would this be better as `acc.extend(xs.filter(|x| x.errno != 0))`?

Reply:
I think this is simpler

Reply:
I'd probably do something like:

```rust
acc.into_iter().chain(xs.into_iter().filter(|x| x.errno != 0)).collect()
```

Which would avoid both `acc` and `xs` from being `mut`.

That said, meh.